generated/lspconfig-servers.json

```diff
@@ -94,7 +94,6 @@
   "emmet_ls": "https://github.com/aca/emmet-ls Package can be installed via `npm`:\n\n``` sh\nnpm install -g emmet-ls\n```",
   "emmylua_ls": "https://github.com/EmmyLuaLs/emmylua-analyzer-rust Emmylua Analyzer\nRust. Language Server for Lua. `emmylua_ls` can be installed using\n`cargo` by following the instructions\\[here\\]\n(https://github.com/EmmyLuaLs/emmylua-analyzer-rust?tab=readme-ov-file#install).\nThe default `cmd` assumes that the `emmylua_ls` binary can be found in\n`$PATH`. It might require you to provide cargo binaries installation\npath in it.",
   "erg_language_server": "https://github.com/erg-lang/erg#flags ELS ELS (erg-language-server) is a\nlanguage server for the Erg programming language. erg-language-server\ncan be installed via `cargo` and used as follows:\n\n``` sh\ncargo install erg --features els\nerg --language-server\n```",
-  "erlangls": "https://erlang-ls.github.io Language Server for Erlang. Clone\n[erlang_ls](https://github.com/erlang-ls/erlang_ls) Compile the project\nwith `make` and copy resulting binaries somewhere in your \\$PATH eg.\n`cp _build/*/bin/* ~/local/bin` Installation instruction can be found\n[here](https://github.com/erlang-ls/erlang_ls). Installation\nrequirements: - [Erlang OTP 21+](https://github.com/erlang/otp) -\n[rebar3 3.9.1+](https://github.com/erlang/rebar3)",
```

Comment:
This file is auto-generated from nvim-lspconfig. You can't just make changes to it directly.

This deprecation should be handled in nvim-lspconfig.